src/api/telephony/portability/document/telephony-portability-document.v6.service.js

```diff
@@ -21,7 +21,7 @@ angular.module('ovh-api-services').service('OvhApiTelephonyPortabilityDocumentV6
     },
     getDocument: {
       method: 'GET',
-      isArray: false,
+      isArray: true,
```

Comment:
`telephony.PortabilityDocument` isn't an Array. Are you sure about this change?

Reply:
Yes I am sure. The call of the API return an array.